src/main/java/teammates/ui/pagedata/FeedbackSubmissionEditPageData.java

```diff
@@ -268,4 +269,12 @@ private FeedbackSubmissionEditQuestion createQuestion(FeedbackQuestionAttributes
 
         return responses;
     }
+
+    private List<FeedbackResponseAttributes> getSortedExistingResponses(FeedbackQuestionAttributes questionAttributes) {
+        List<FeedbackResponseAttributes> existingResponses = bundle.questionResponseBundle.get(questionAttributes);
+        Map<String, String> emailNamePair = this.bundle.getSortedRecipientList(questionAttributes.getFeedbackQuestionId());
+        existingResponses.sort(Comparator.comparing(existingResponse -> emailNamePair.get(existingResponse.recipient)));
```

Comment:
I debugged and found out that tests are failing because this line gives `Null Pointer Exception`.

The logs: 
![image](https://user-images.githubusercontent.com/12574756/37888150-4b526116-30f8-11e8-9ddb-3b69579a04bd.png)